meinberlin/apps/plans/templates/meinberlin_plans/plan_update_form.html

```diff
@@ -84,9 +80,8 @@ <h1 class="u-first-heading">{% trans 'Edit plan' %}</h1>
         {% endif %}
     </div>
 </div>
-{% endblock %}
 
-<div class="modal" tabindex="-1" role="dialog" id="plan-publish-modal" aria-labelledby="plan-publish-label">
+<div class="modal" tabindex="-1" role="dialog" id="plan-publish-modal" aria-labelledby="plan-publish-label" aria-hidden="true">
```

Comment:
this is got from the docs, I assume it's incase someones screen reader is just reading everything weather it's open or not? it works fine anyway

Reply:
Sorry, only registered with me when rereading in my mails. Should we check the publish button for the projects for a11y as well?